lib/plugins/aws/provider.js

```diff
@@ -1779,6 +1803,21 @@ class AwsProvider {
     return runtime || runtimeSourceValue.value || defaultRuntime;
   }
 
+  resolveFunctionRuntimeManagement(functionRuntimeManagement) {
+    const defaultRuntimeManagement = {
+      mode: 'auto',
+    };
+    const resolvedFunctionRuntimeManagement = resolveRuntimeManagement(functionRuntimeManagement);
+    const resolvedProviderRuntimeManagement = resolveRuntimeManagement(
+      this.serverless.service.provider.runtimeManagement
+    );
+    return {
+      ...defaultRuntimeManagement,
```

Comment:
Let's not introduce that variable, just write: `mode: 'auto',` inline.

Similarly creating other variables is controversial, as there's just single use of them, so resolution can be done on spot

Reply:
Updated to remove all 3 consts